invenio_vocabularies/contrib/awards/schema.py

```diff
@@ -9,26 +9,90 @@
 """Awards schema."""
 
 from flask_babelex import lazy_gettext as _
-from marshmallow import Schema, ValidationError, fields, validate, \
-    validates_schema
-from marshmallow_utils.fields import SanitizedUnicode
+from functools import partial
+from marshmallow import Schema, ValidationError, fields, post_load, pre_dump, \
+    pre_load, validate, validates_schema
+from marshmallow_utils.fields import IdentifierSet, SanitizedUnicode
+from marshmallow_utils.schemas import IdentifierSchema
 
+from .config import award_schemes
 from ..funders.schema import FunderRelationSchema
+from ...services.schema import BaseVocabularySchema, i18n_strings
 
 
-class AwardRelationSchema(Schema):
+class AwardSchema(BaseVocabularySchema):
     """Award schema."""
-
-    title = SanitizedUnicode(
-        required=True,
-        validate=validate.Length(min=1, error=_('Title cannot be blank.'))
-    )
+    identifiers = IdentifierSet(fields.Nested(
+        partial(
+            IdentifierSchema,
+            allowed_schemes=award_schemes,
+            identifier_required=False
+        )
+    ))
     number = SanitizedUnicode(
         required=True,
         validate=validate.Length(min=1, error=_('Number cannot be blank.'))
```

Comment:
minor: is this needed? does the required check for "not blank"? I guess it just checks its presence?